### DISCUSSION TITLE
How can I access other vendors' stores for draft application?

### DISCUSSION THREAD

**Discussion by cnr91** [amzn/selling-partner-api-models#3542] [General]
How can I access other vendors' stores for draft application?

I have activated the application as beta on other vendors. But I don't have **MWS token** for vendor, I just have **spapi_oauth_code** and **seller id**.

In this situation, how should I proceed so that I can access the seller's information?

![draft-sp-api](https://user-images.githubusercontent.com/75105673/223734115-a5678826-a0c4-40fc-bb72-0004bacf2935.jpg)

**Comment by sungolivia** [comment]
Hey @cnr91 so I don't think you can access your vendor's accounts, I would suggest reaching out to your customers so you will get those authorizations!

**Reply by cnr91** [reply]
Thank you for your answer. 
My question is I am giving permission for another store in the beta version. 
So it actually allows as OAuthorisations a store, not as vendors.
But in this case, I cannot reach the user's store. Access is denied when I try to access its data.

We can change the question as follows. 
How can I access the data of the store for which I have obtained OAuthorisations permission?

**Reply by MeenaAmz** [reply]
Once you get the oauth permission from your vendor, you will received the authorization code which you can exchange for LWA refresh token using [this](https://developer-docs.amazon.com/sp-api/docs/website-authorization-workflow#step-4-your-application-exchanges-the-lwa-authorization-code-for-a-lwa-refresh-token). Once you have the LWA refresh token follow these [steps](https://developer-docs.amazon.com/sp-api/docs/connecting-to-the-selling-partner-api) to get connected to SP-API
